code/modules/admin/admin_verbs.dm

```diff
@@ -370,6 +385,135 @@ GLOBAL_LIST_INIT(view_runtimes_verbs, list(
 		// TODO: SStgui.on_transfer() to move windows from old and new
 		SSblackbox.record_feedback("tally", "admin_verb", 1, "Aghost") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
+/client/proc/admin_ghost()
+	set category = "Admin"
+	set name = "Aghost"
+
+	if(!check_rights(R_ADMIN|R_MOD))
+		return
+
+	do_aghost()
+
+/// Allow an admin to observe someone.
+/// mentors are allowed to use this verb while living, but with some stipulations:
+/// if they attempt to do anything that would stop their orbit, they will immediately be returned to their body.
```

Comment:
does auto doc allow for this kind of thing?

Reply:
mhm, it just combines them into a single line.